FWCore/Skeletons/mkTemplates/ESProducer/ESProducer.cc

```diff
@@ -72,53 +73,46 @@ class __class__ : public edm::ESProducer {
 //
 // constructors and destructor
 //
-__class__::__class__(const edm::ParameterSet& iConfig)
-{
-   //the following line is needed to tell the framework what
-   // data is being produced
-   setWhatProduced(this);
+__class__::__class__(const edm::ParameterSet& iConfig) {
+  //the following line is needed to tell the framework what
+  // data is being produced
+  setWhatProduced(this);
```

Comment:
Should we add an example of how to use consumes?

Reply:
Probably, thanks.

Reply:
Let me add that if we manage to get this PR in quickly (i.e. past code checks), I'll leave the ESProducer consumes to a subsequent PR.